### ISSUE TITLE
Seems like period/dot in filename breaks the GH action

### ISSUE THREAD

**Issue by praiskup** [sanitizers/chronographer-github-app#44]
The `leap-15.4.config` has not been detected: https://github.com/rpm-software-management/mock/pull/1464

**Comment by webknjaz** [comment]
Technically, that's expected due to the semantics of how Towncrier itself perceives the fragments. You probably need to fix your usage of Towncrier. Either replace the period with an underscore or a dash, or perhaps even go for an orphan fragment (starting with a leading `+` which is special-cased in Towncrier).
Usually, the first part is the issue/PR number to be linked. Why didn't you want to follow that format?

**Comment by praiskup** [comment]
I opened this, because we hit a real-life issue with towncrier:

```
$ towncrier create foo.bar.baz.config
Created news fragment at /home/praiskup/rh/projects/mock/mock/releng/release-notes-next/foo.bar.baz.config
```

At least we were allowed to create such file, I assumed that it "is a correct snippet" then, and I'd expect that the action accepts it as well.  But is this a bug of `towncrier create`?

**Comment by webknjaz** [comment]
@praiskup yes, I'm sure it's a bug in `towncrier create`. Though, I've never used that command since I have a habit of just creating these text files on my own. Knowing how Towncrier renders the fragments, I'd say that it would not identify such a file correctly.

The docs aren't very verbose on what happens with multiple periods: https://towncrier.rtfd.io/en/stable/tutorial.html#creating-news-fragments. But when I last read the source code, it was clear that the identifier is what's before the first period. Then, there's a category before the next optional period. After then, there's an optional ordinary number and an arbitrary suffix.

So, when building, Towncrier will parse `foo.bar.baz.config` as having `foo` for identifier (issue/PR reference normally), `bar` as a category and will probably discard `.baz.config` (or maybe error out because that doesn't start with a number?)

**Comment by webknjaz** [comment]
I've filed https://github.com/twisted/towncrier/issues/733 upstream. We no longer need to track this question in the bot repo.
